auth/token-managers/token-manager.ts

```diff
@@ -229,7 +229,7 @@ export class TokenManager {
    * Checks if currently-stored token should be refreshed
    * i.e. past the window to request a new token
    */
-  private tokenNeedsRefresh(): boolean {
+  protected tokenNeedsRefresh(): boolean {
```

Comment:
technically, I didn't need to override `tokenNeedsRefresh()` but it didn't seem right leaving it as the only private function :)